packages/server/scripts/integrations/postgres/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3.8"
 services:
   db:
     container_name: postgres
-    image: postgres:15-bullseye
+    image: postgres:latest
```

Comment:
Is this strictly necessary? Seems risky.

Reply:
this is just one of our manual scripts for testing, it was always latest before but had to be changed due to issues with Linux+Docker compose compat issues, those seem to have been resolved so can go back to latest.